sql/mysqld.cc

```diff
@@ -4642,7 +4641,7 @@ static int init_ssl()
 {
 #ifdef HAVE_OPENSSL
 #ifndef HAVE_YASSL
-  CRYPTO_malloc_init();
+  OPENSSL_malloc_init();
```

Comment:
If I read upstream code correctly, it doesn't call `OPENSSL_malloc_init` with OpenSSL 1.1.

Reply:
`CRYPTO_malloc_init` was renamed `OPENSSL_malloc_init`, so this seems like the most correct porting. On the other hand, it's not a very useful function, so I'll just drop it.